### PR TITLE
Fixed #24455 -- Fixed crash in debug view with lazy objects

### PR DESCRIPTION
**Pull request by bpeschier**
Refs #24455

### CODE REVIEW

File: tests/view_tests/tests/test_debug.py [hunk 1]
```diff
@@ -380,6 +381,30 @@ def test_unfrozen_importlib(self):
         html = reporter.get_traceback_html()
         self.assertIn('<h1>ImportError at /test_view/</h1>', html)
 
+    def test_ignore_traceback_evaluation_exceptions(self):
+        """
+        Don't trip over exceptions generated by crafted objects when
+        evaluating them while cleansing. Refs #24455.
+        """
+        request = self.rf.get('/test_view/')
+        broken_lazy = SimpleLazyObject(lambda: 0 / 0)
```
Comment (bmispelon) [inline tests/view_tests/tests/test_debug.py]: I think we should trigger a custom exception here rather than `ZeroDivisionError`

**Comment by bmispelon** [comment]
Code looks correct to me.

Can you add a mention in the 1.9 release notes?
And while you're at it, can you reformat the commit message using our guidelines (https://docs.djangoproject.com/en/dev/internals/contributing/committing-code/#committing-guidelines)?

Dankjewel!

File: django/views/debug.py [hunk 2]
```diff
@@ -186,7 +186,15 @@ def get_post_parameters(self, request):
                 return request.POST
 
     def cleanse_special_types(self, request, value):
-        if isinstance(value, HttpRequest):
+        try:
+            # If value is lazy or a crafted complex object of another kind,
+            # this check might raise an exception. isinstance checks for lazy
```
Comment (timgraham) [inline django/views/debug.py]: checks that

**Comment by bmispelon** [comment]
Maybe it doesn't need a release note after all.

Can you just force-push the commit message thingy, thanks?

**Comment by timgraham** [comment]
buildbot, test this please.

**Comment by timgraham** [comment]
merged in 756cee46d25a52108fd7190f787a7e636962df5d, thanks!

**Pull request closed without merging**